<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KMCTGLT" failures="1" tests="2" name="Tata_EMS_Test" time="43.539" errors="0" timestamp="2023-02-21T18:51:34 IST">
  <testcase classname="tata_EMS_Project_Test_Cases.Login_Test" name="Test01_Verifying_login_with_InValidDetails" time="1.543">
    <failure type="java.lang.AssertionError" message="expected [https://staging.netscribes.co/tata_ems_testing/admin/d2b5316eadb139ea4a091aeca9ec5bd3] but found [https://staging.netscribes.co/tata_ems_testing/admin/dashboard]">
      <![CDATA[java.lang.AssertionError: expected [https://staging.netscribes.co/tata_ems_testing/admin/d2b5316eadb139ea4a091aeca9ec5bd3] but found [https://staging.netscribes.co/tata_ems_testing/admin/dashboard]
at tata_EMS_Project_Test_Cases.Login_Test.Test01_Verifying_login_with_InValidDetails(Login_Test.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Test01_Verifying_login_with_InValidDetails -->
  <testcase classname="tata_EMS_Project_Test_Cases.Login_Test" name="Test01_Verifying_login_with_ValidDetails" time="4.255"/>
</testsuite> <!-- Tata_EMS_Test -->
